### ISSUE TITLE
‘vagrant up’ error

### ISSUE THREAD

**Issue by AqgRVQEP** [ValveSoftware/Proton#2766]
```
proton git:(proton_4.2) ✗ sudo vagrant up                                      
[sudo] kyle 的密码： 
Platform: 12 CPUs, 7947 MB memory
Bringing machine 'default' up with 'virtualbox' provider...
#<Thread:0x0000561735721170@/usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/batch_action.rb:71 run> terminated with exception (report_on_exception is true):
Traceback (most recent call last):
	25: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/batch_action.rb:82:in `block (2 levels) in run'
	24: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/machine.rb:188:in `action'
	23: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/machine.rb:188:in `call'
	22: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/environment.rb:592:in `lock'
	21: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/machine.rb:202:in `block in action'
	20: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/machine.rb:227:in `action_raw'
	19: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/runner.rb:66:in `run'
	18: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/util/busy.rb:19:in `busy'
	17: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/runner.rb:66:in `block in run'
	16: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/builder.rb:116:in `call'
	15: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/warden.rb:34:in `call'
	14: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/plugins/providers/virtualbox/action/check_virtualbox.rb:17:in `call'
	13: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/warden.rb:34:in `call'
	12: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/builtin/call.rb:53:in `call'
	11: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/runner.rb:66:in `run'
	10: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/util/busy.rb:19:in `busy'
	 9: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/runner.rb:66:in `block in run'
	 8: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/builder.rb:116:in `call'
	 7: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/warden.rb:34:in `call'
	 6: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
	 5: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/warden.rb:34:in `call'
	 4: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/builtin/handle_box.rb:56:in `call'
	 3: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/warden.rb:34:in `call'
	 2: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
	 1: from /usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/warden.rb:34:in `call'
/usr/share/rubygems-integration/all/gems/vagrant-2.0.2/lib/vagrant/action/builtin/config_validate.rb:18:in `call': There are errors in the configuration of this machine. Please fix (Vagrant::Errors::ConfigInvalid)
the following errors and try again:

vagrant:
* The following settings shouldn't exist: plugins

There are errors in the configuration of this machine. Please fix
the following errors and try again:

vagrant:
* The following settings shouldn't exist: plugins
```

but i'm already  install vagrant-sshfs
```
sudo apt install vagrant-sshfs     
正在读取软件包列表... 完成
正在分析软件包的依赖关系树       
正在读取状态信息... 完成       
vagrant-sshfs 已经是最新版 (1.3.0-2)。
```

**Comment by mrpippy** [comment]
Commenting out the `config.vagrant.plugins` line in the `Vagrantfile` seemed to resolve the issue. I'm on Ubuntu 18.04.

**Comment by madewokherd** [comment]
Does /vagrant still correctly point to the vagrant_share directory on the host machine? I would expect it to be necessary to edit that share to not use sshfs.

**Comment by mrpippy** [comment]
Yes, `/vagrant` seems to be mounted correctly and created files show up on the host machine

**Comment by aeikum** [comment]
That's just a hint to Vagrant to make sure the plugin is installed. If it's installed already, then you don't need that line. It seems this variable was added in a recent Vagrant version. Can you get a more recent version from your distro?

**Comment by mrpippy** [comment]
Yeah that's the problem, `config.vagrant.plugins` was added in 2.2.0 but Ubuntu 18.04 includes 2.0.3. The best fix is probably to add a `Vagrant.require_version ">= 2.2.0"` line to the Vagrantfile to ensure a supported version is used. I'll make a PR for it.

**Comment by aeikum** [comment]
Should be fixed by ee5de7e3be2ec9bcca5c28969b3d47b035b991b1. Please upgrade your Vagrant installation.
